internal/api/v1/application/part.go

```diff
@@ -87,9 +87,11 @@ func GetPart(c *gin.Context) apierror.APIErrors {
 		return fetchAppImage(c, ctx, logger, cluster, app.Meta)
 	case "values":
 		return fetchAppValues(c, logger, cluster, app.Meta)
+	case "manifest":
+		return fetchAppManifest(c, app)
 	}
 
-	return apierror.NewBadRequestError("unknown part, expected chart, image, or values")
+	return apierror.NewBadRequestError("unknown part, expected chart, image, manifest, or values")
```

Comment:
This code should never be reached (we already checked the part). It's probably better to return a `500` (?).

Reply:
Done, commit 281c55964128a18f8a6bcf3b4d75b06e80395032